_rules/SC2-4-2-page-has-title.md

```diff
@@ -22,19 +22,19 @@ authors:
 
 ### Applicability
 
-The rule applies to any page where the root element is an `html` element, and which is not embedded in another page.
+The rule applies to any page where the [document element](https://www.w3.org/TR/dom/#document-element) is an `html` element, and where the page is not embedded in another page.
 
-**Note**: Pages may be embedded inside other pages through elements such as iframes and object elements.
+**Note**: Pages may be embedded inside other pages through elements such as `iframe` and `object` elements.
 
 ### Expectation 1
 
-The page contains at least one `title` element.
+The [document element](https://www.w3.org/TR/dom/#document-element) has at least one [descendant](https://www.w3.org/TR/dom41/#concept-tree-descendant) that is an HTML `title` element.
 
-**Note**: The `title` element exists in other namespaces such as SVG. These are not `title` elements for HTML document and should be ignored.
+**Note**: The `title` element exists in other namespaces such as SVG. These are not HTML `title` elements and should be ignored for this rule.
 
 ### Expectation 2
 
-The first `title` element contains [non-empty text](#non-empty).
+The first HTML `title` element that is a [descendant](https://www.w3.org/TR/dom41/#concept-tree-descendant) of the [document element](https://www.w3.org/TR/dom/#document-element) has [children](https://www.w3.org/TR/dom/#concept-tree-child) that are [text nodes](https://www.w3.org/TR/dom/#text) that are not only [whitespace](#whitespace).
```

Comment:
@kasperisager, should it have been specified whether the `title` element we are looking for is a descendant in the flat tree or something else?

Reply:
That’s actually a really good point. In this, we’re _not_ looking for a descendant in the flat tree, which is not something that has been made explicitly previously. We’ve only made it explicit when we’re specifically looking for either a shadow-including descendant or a descendant in the flat tree. What are people’s thoughts on this? Is an unqualified use of the term “descendant” unambiguous? I’m partial to thinking that it is.

Reply:
@kasperisager and I have agreed among ourselves that we feel the wording is fine. So unless someone else disagrees, we will leave it as is.